config/application.rb

```diff
@@ -47,6 +47,9 @@ class Application < Rails::Application
     # NOTE: This didn't work when put in config/initializers/responders.rb.
     config.responders.flash_keys = [:success, :error]
 
+    # Use per form CSRF tokens for all controllers
+    Rails.configuration.action_controller.per_form_csrf_tokens = true
```

Comment:
This is already loaded by default on line 17 of this file (try looking at the source of load_defaults), it's not necessary.